packages/frontend/src/components/DashboardFilter/FilterConfiguration/TileFilterConfiguration.tsx

```diff
@@ -36,6 +40,8 @@ type Props = {
 
 const TileFilterConfiguration: FC<Props> = ({
     tiles,
+    tabs,
+    activeTabUuid,
```

Comment:
are these 2 props used ?

Reply:
yes, 

tabs are used to create the Accordion panels and activeTabUUid is to open the Accordion panel if tabuuid === activeTabUuid